keyboards/durgod/k3x0/readme.md

```diff
@@ -18,34 +18,58 @@ Instructions for building the K310 and K320 firmware can be found here:
 
-1. Back up original Firmware Image:
-- Unplug
-- Short Boot0 to Vdd (See below)
-- Plug In USB
-- Make a Flash Image's Backup in case you wanted to restore the Keyboard to Stock's Image:
-  - Using dfu-util:
+- The Durgod K320/K310 - different batches have different controllers, so when opening the keyboard, you will have to make sure it's using SMT32 controller.
+- Another keyboard - when on bootloader mode, you can't use the keyboard to type and execute the commands (not really required if you can use any mouse interface to type and execute the commands).
+- 2 prying tools - to open the keyboard.
+- Another helping hand - it can be also done with just two hands, but it will require a lot of dexterity.
+- A small cable with both ends cut.
 
-        dfu-util --list
-        dfu-util --alt 0 --dfuse-address 0x08000000 --upload ${OLD_STOCK_BIN}
+#### Enter bootloader while using the original firmware
 
-2. Flash the QMK Firmware Image.
-- Put board into Bootloader mode, using the same method as when backing up the original Firmware (above)
-- Here are a few options for performing the initial Flash:
-  - Using [QMK Toolbox](https://github.com/qmk/qmk_toolbox)
+1. Unplug USB cable.
+2. Open the keyboard case - there are plenty of youtube videos in that, but you will basically need 2 prying tools and a certain amount of force.
+3. After opening the keyboard, put the PCB upsides down, and the controller should be near the LEDs, and make sure it's using the right controller (SMT32).
+4. Use the cable to short Boot0 to Vdd, by shorting R21 to C27 on the sides closest to the processor, as shown:
+
+<img src="https://i.imgur.com/hvDnw5a.jpg" width="520" height="693" alt="Shorting Boot0 to Vdd on K320">
+
+5. While shorting both contacts as instructed in step 4, plug in the USB cable.
+6. When the USB cable is fully inserted, the LEDs should not turn on and the keyboard it's not able to type, you are probably on bootloader mode, to be sure:
+  - Using dfu-util, it should show something like "Found DFU…" after using this command:
+```
+dfu-util --list
+```
+
+Now that you are sure to be in bootloader mode, you can backup the original firmware and flash QMK Firmware.
+
+If it does not behave like described, remove the cable and go back to step 4.
+
+#### Backup original firmware
+
+1. Be in Bootloader mode.
+2. Execute this command to backup:
   - Using dfu-util:
+```bash
+dfu-util -a 0 -d 0483:DF11 -s 0x08000000 -U k3x0_original.bin
+```
 
-        dfu-util --alt 0 --dfuse-address 0x08000000 --download ${NEW_QMK_BIN}
+#### Flash the QMK Firmware Image
 
-You can short Boot0 to Vdd by shorting R21 to C27 on the sides closest to the processor, as shown:
+1. Be in Bootloader mode.
+2. Here are a few options for performing the initial Flash:
+  - Using [QMK Toolbox](https://github.com/qmk/qmk_toolbox)
+  - Using qmk on command line:
 
-<img src="https://i.imgur.com/hvDnw5a.jpg" width="520" height="693" alt="Shorting Boot0 to Vdd on K320">
+```bash
+qmk flash
```

Comment:
```suggestion
qmk flash -kb durgod/k3x0 -km default
```

Reply:
using only `durgod/k3x0` will always build k310 file, so i put both commands for each variation.